test/integration/page-objects/menus.js

```diff
@@ -0,0 +1,80 @@
+    return browser.sleep(300).then(() => {
+      return element(childMenuSelector).click();
+    });
+  });
+};
+
+exports.menus = {
+  topMenu: {
+    clickDocumentation: () => {
+      return clickNestedMenuItem(by.id('help-dropdown'), by.cssContainingText('.dropdown.open', 'Documentation'));
+    },
+    clickCLI: () => {
+      return clickNestedMenuItem(by.id('help-dropdown'), by.css('.dropdown.open', 'Command Line Tools'));
+    },
+    clickAbout: () => {
+      return clickNestedMenuItem(by.id('help-dropdown'), by.css('.dropdown.open', 'About'));
+    },
+    clickLogout: () => {
+      return clickNestedMenuItem(by.id('iser-dropdown'), by.css('.dropdown.open', 'Logout'));
+    }
+  },
+  // example:
+  //  AnyPage.sideMenu.clickPods();
+  sideMenu: {
+    clickOverview: () => {
+      return element(by.cssContainingText('.dropdown-toggle', 'Overview').row(0)).click();
+    },
+    // applications submenu
+    clickDeployments: () => {
+      return clickNestedMenuItem(by.cssContainingText('.dropdown-toggle', 'Applications'), by.cssContainingText('a', 'Deployments'));
+    },
+    clickPods: () => {
+      return clickNestedMenuItem(by.cssContainingText('.dropdown-toggle', 'Applications'), by.cssContainingText('a', 'Pods'));
+    },
+    clickServices: () => {
+      return clickNestedMenuItem(by.cssContainingText('.dropdown-toggle', 'Applications'), by.cssContainingText('a', 'Services'));
+    },
+    clickRoutes: () => {
+      return clickNestedMenuItem(by.cssContainingText('.dropdown-toggle', 'Applications'), by.cssContainingText('a', 'Routes'));
+    },
+    // builds submenu
+    clickBuilds: () => {
+      return clickNestedMenuItem(by.cssContainingText('.dropdown-toggle', 'Builds'), by.cssContainingText('a', 'Builds'));
+    },
+    clickPipelines: () => {
+      return clickNestedMenuItem(by.cssContainingText('.dropdown-toggle', 'Builds'), by.cssContainingText('a', 'Pipelines'));
+    },
+    clickImages: () => {
+      return clickNestedMenuItem(by.cssContainingText('.dropdown-toggle', 'Builds'), by.cssContainingText('a', 'Images'));
+    },
+    // resources submenu
+    clickQuota: () => {
+      return clickNestedMenuItem(by.cssContainingText('.dropdown-toggle', 'Resources'), by.cssContainingText('a', 'Quota'));
+    },
+    clickMembership: () => {
+      return clickNestedMenuItem(by.cssContainingText('.dropdown-toggle', 'Resources'), by.cssContainingText('a', 'Membership'));
+    },
+    clickConfigMaps: () => {
+      return clickNestedMenuItem(by.cssContainingText('.dropdown-toggle', 'Resources'), by.cssContainingText('a', 'Config Maps'));
+    },
+    clickSecrets: () => {
+      return clickNestedMenuItem(by.cssContainingText('.dropdown-toggle', 'Resources'), by.cssContainingText('a', 'Secrets'));
+    },
+    clickOtherResources: () => {
+      return clickNestedMenuItem(by.cssContainingText('.dropdown-toggle', 'Resources'), by.cssContainingText('a', 'Other Resources'));
+    },
+    // rest
```

Comment:
I'll update to `// the rest of the top lvl menu items`.  It may have been good to just nest these for clarity, ie `leftNav.apps.clickDeployment()`, `leftNav.builds.clickPipelines()`.  I'm open.